cmake/Corrosion.cmake

```diff
@@ -120,6 +120,7 @@ if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.19.0)
     set(_CORR_PROP_NO_DEFAULT_FEATURES CORROSION_NO_DEFAULT_FEATURES CACHE INTERNAL "")
     set(_CORR_PROP_ENV_VARS CORROSION_ENVIRONMENT_VARIABLES CACHE INTERNAL "")
     set(_CORR_PROP_HOST_BUILD CORROSION_USE_HOST_BUILD CACHE INTERNAL "")
+    set(_CORR_PROP_FLAGS CORROSION_CARGO_FLAGS CACHE INTERNAL "")
```

Comment:
This if/else block was added for compatibilty reasons to not have to rename existing property names.
For new properties don't extend this if else block. Insteasd just use `INTERFACE_CORROSION_CARGO_FLAGS` as the property name. The `INTERFACE_` prefix is required to support CMake versions before 3.19.